internal/handlers/pg/pgdb/pool_test.go

```diff
@@ -198,41 +198,57 @@ func TestConcurrentCreate(t *testing.T) {
 	schemaName := testutil.SchemaName(t)
 	tableName := testutil.TableName(t)
 
-	for _, withTable := range []bool{false, true} {
-		start := make(chan struct{})
-		res := make(chan error, n)
-		for i := 0; i < n; i++ {
-			go func() {
-				<-start
-				if withTable {
-					res <- pool.CreateTable(ctx, schemaName, tableName)
-				} else {
-					res <- pool.CreateSchema(ctx, schemaName)
-				}
-			}()
-		}
+	for _, tc := range []struct {
+		name        string
+		f           func() error
+		compareFunc func(*testing.T, int) bool
+	}{
+		{
+			name: "CreateDatabase",
+			f: func() error {
+				return pool.CreateDatabase(ctx, schemaName)
+			},
+			compareFunc: func(t *testing.T, errors int) bool {
+				return assert.Equal(t, n-1, errors)
+			},
+		}, {
+			name: "CreateCollection",
+			f: func() error {
+				return pool.CreateCollection(ctx, schemaName, tableName)
```

Comment:
Is this `tableName` should we a `collectionName` now?

Reply:
Yeah, I suggest working on those tests in a separate PR. We might want to add more tests for the Pool.

Reply:
Ha, I did not read it before review. Good to see us on the same page